dateparser/data/languages_info.py

```diff
@@ -206,6 +206,630 @@
+        "lb"
+    ],
+    "lg": [
+        "lg"
+    ],
+    "lkt": [
+        "lkt"
+    ],
+    "ln": [
+        "ln"
+    ],
+    "lo": [
+        "lo"
+    ],
+    "lrc": [
+        "lrc"
+    ],
+    "lt": [
+        "lt"
+    ],
+    "lu": [
+        "lu"
+    ],
+    "luo": [
+        "luo"
+    ],
+    "luy": [
+        "luy"
+    ],
+    "lv": [
+        "lv"
+    ],
+    "mas": [
+        "mas"
+    ],
+    "mer": [
+        "mer"
+    ],
+    "mfe": [
+        "mfe"
+    ],
+    "mg": [
+        "mg"
+    ],
+    "mgh": [
+        "mgh"
+    ],
+    "mgo": [
+        "mgo"
+    ],
+    "mi": [
+        "mi"
+    ],
+    "mk": [
+        "mk"
+    ],
+    "ml": [
+        "ml"
+    ],
+    "mn": [
+        "mn"
+    ],
+    "mr": [
+        "mr"
+    ],
+    "ms": [
+        "ms"
+    ],
+    "mt": [
+        "mt"
+    ],
+    "mua": [
+        "mua"
+    ],
+    "my": [
+        "my"
+    ],
+    "mzn": [
+        "mzn"
+    ],
+    "naq": [
+        "naq"
+    ],
+    "nb": [
+        "nb"
+    ],
+    "nd": [
+        "nd"
+    ],
+    "ne": [
+        "ne"
+    ],
+    "nl": [
+        "nl"
+    ],
+    "nmg": [
+        "nmg"
+    ],
+    "nn": [
+        "nn"
+    ],
+    "nnh": [
+        "nnh"
+    ],
+    "nus": [
+        "nus"
+    ],
+    "nyn": [
+        "nyn"
+    ],
+    "om": [
+        "om"
+    ],
+    "or": [
+        "or"
+    ],
+    "os": [
+        "os"
+    ],
+    "pa": [
+        "pa",
+        "pa-Arab",
+        "pa-Guru"
+    ],
+    "pl": [
+        "pl"
+    ],
+    "ps": [
+        "ps"
+    ],
+    "pt": [
+        "pt"
+    ],
+    "qu": [
+        "qu"
+    ],
+    "rm": [
+        "rm"
+    ],
+    "rn": [
+        "rn"
+    ],
+    "ro": [
+        "ro"
+    ],
+    "rof": [
+        "rof"
+    ],
+    "ru": [
+        "ru"
+    ],
+    "rw": [
+        "rw"
+    ],
+    "rwk": [
+        "rwk"
+    ],
+    "sah": [
+        "sah"
+    ],
+    "saq": [
+        "saq"
+    ],
+    "sbp": [
+        "sbp"
+    ],
+    "sd": [
+        "sd"
+    ],
+    "se": [
+        "se"
+    ],
+    "seh": [
+        "seh"
+    ],
+    "ses": [
+        "ses"
+    ],
+    "sg": [
+        "sg"
+    ],
+    "shi": [
+        "shi",
+        "shi-Latn",
+        "shi-Tfng"
+    ],
+    "si": [
+        "si"
+    ],
+    "sk": [
+        "sk"
+    ],
+    "sl": [
+        "sl"
+    ],
+    "smn": [
+        "smn"
+    ],
+    "sn": [
+        "sn"
+    ],
+    "so": [
+        "so"
+    ],
+    "sq": [
+        "sq"
+    ],
+    "sr": [
+        "sr",
+        "sr-Cyrl",
+        "sr-Latn"
+    ],
+    "sv": [
+        "sv"
+    ],
+    "sw": [
+        "sw"
+    ],
+    "ta": [
+        "ta"
+    ],
+    "te": [
+        "te"
+    ],
+    "teo": [
+        "teo"
+    ],
+    "tg": [
+        "tg"
+    ],
+    "th": [
+        "th"
+    ],
+    "ti": [
+        "ti"
+    ],
+    "tl": [
+        "tl"
+    ],
+    "to": [
+        "to"
+    ],
+    "tr": [
+        "tr"
+    ],
+    "tt": [
+        "tt"
+    ],
```

Comment:
```suggestion
    "tt": [
        "tt"
    ],
```

Reply:
Hi @gavishpoddar, if I'm not wrong you forgot to apply this one. I reopened it.

Reply:
Hi @noviluni, I am unable to apply this as GitHub is showing the below error

`Suggestion cannot be identical to original text. 
`

Reply:
Ok, this suggestion is wrong, for some reason is adding `tt` instead of removing it :thinking: I created a new different suggestion removing `tt`.